packages/vats/src/proposals/upgrade-paRegistry-proposal.js

```diff
@@ -51,6 +54,9 @@ export const getManifestForUpgradingRegistry = (_powers, { registryRef }) => ({
       brand: { consume: { [Stable.symbol]: par } },
     },
   },
+  installations: {
+    priceAuthority: restoreRef(registryRef),
```

Comment:
```suggestion
    priceAggregator: restoreRef(registryRef),
```

Reply:
wait... now I'm getting things mixed up.

priceAuthorityRegistry is not a contract vat.

https://github.com/Agoric/agoric-sdk/blob/a92cf556a0c806df293027458883f2a81a2bfade/packages/vats/src/vat-priceAuthority.js